languages.go

```diff
@@ -0,0 +1,1208 @@
+		return "Ндонга"
+	case LanguageNL:
+		return "Голландский"
+	case LanguageNN:
+		return "Норвежский нюнорск"
+	case LanguageNO:
+		return "Норвежский"
+	case LanguageNS:
+		return "Северный сото"
+	case LanguageNV:
+		return "Навахо"
+	case LanguageNY:
+		return "Чичева"
+	case LanguageOC:
+		return "Окситанский"
+	case LanguageOJ:
+		return "Оджибве"
+	case LanguageOM:
+		return "Оромо"
+	case LanguageOR:
+		return "Ория"
+	case LanguageOS:
+		return "Осетинский"
+	case LanguagePA:
+		return "Панджаби"
+	case LanguagePI:
+		return "Пали"
+	case LanguagePL:
+		return "Польский"
+	case LanguagePS:
+		return "Пушту"
+	case LanguagePT:
+		return "Португальский"
+	case LanguageQU:
+		return "Кечуа"
+	case LanguageRM:
+		return "Ретороманский"
+	case LanguageRN:
+		return "Рунди"
+	case LanguageRO:
+		return "Румынский"
+	case LanguageRU:
+		return "Русский"
+	case LanguageRW:
+		return "Киньяруанда"
+	case LanguageSA:
+		return "Санскрит"
+	case LanguageSC:
+		return "Сардинский"
+	case LanguageSD:
+		return "Синдхи"
+	case LanguageSE:
+		return "Северносаамский"
+	case LanguageSG:
+		return "Санго"
+	case LanguageSI:
+		return "Сингальский"
+	case LanguageSK:
+		return "Словацкий"
+	case LanguageSL:
+		return "Словенский"
+	case LanguageSM:
+		return "Самоанский"
+	case LanguageSN:
+		return "Шона"
+	case LanguageSO:
+		return "Сомалийский"
+	case LanguageSQ:
+		return "Албанский"
+	case LanguageSR:
+		return "Сербский"
+	case LanguageSS:
+		return "Свази"
+	case LanguageST:
+		return "Сесото"
+	case LanguageSU:
+		return "Сунданский"
+	case LanguageSV:
+		return "Шведский"
+	case LanguageSW:
+		return "Суахили"
+	case LanguageTA:
+		return "Тамильский"
+	case LanguageTE:
+		return "Телугу"
+	case LanguageTG:
+		return "Таджикский"
+	case LanguageTH:
+		return "Тайский"
+	case LanguageTI:
+		return "Тигринья"
+	case LanguageTK:
+		return "Туркменский"
+	case LanguageTL:
+		return "Тагальский"
+	case LanguageTN:
+		return "Тсвана"
+	case LanguageTO:
+		return "Тонганский"
+	case LanguageTR:
+		return "Турецкий"
+	case LanguageTS:
+		return "Тсонга"
+	case LanguageTT:
+		return "Татарский"
+	case LanguageTW:
+		return "Тви"
+	case LanguageTY:
+		return "Таитянский"
+	case LanguageUG:
+		return "Уйгурский"
+	case LanguageUK:
+		return "Украинский"
+	case LanguageUR:
+		return "Урду"
+	case LanguageUZ:
+		return "Узбекский"
+	case LanguageVE:
+		return "Венда"
+	case LanguageVI:
+		return "Вьетнамский"
+	case LanguageVO:
+		return "Волапюк"
+	case LanguageWA:
+		return "Валлонский"
+	case LanguageWO:
+		return "Волоф"
+	case LanguageXH:
+		return "Коса"
+	case LanguageYI:
+		return "Идиш"
+	case LanguageYO:
+		return "Йоруба"
+	case LanguageZA:
+		return "Чжуанский"
+	case LanguageZH:
+		return "Китайский"
+	case LanguageZU:
+		return "Зулу"
+	default:
+		return "Неизвестный"
+	}
+}
+
+func (c LanguageCode) IsValid() bool {
+	return c.String() != UnknownLanguageCode
+}
+
+func TotalLanguages() int {
+	return 184 // Including LanguageUnknown
```

Comment:
Why not a `len(AllLanguages())` ?

I'm afraid this number won't be updated, also, what is the need behind such function?

Reply:
i'm not quite sure, i just did it the way that it's already done in the other types.

Reply:
Resolved